ui-react/packages/atlasmap-core/src/services/mapping-management.service.ts

```diff
@@ -31,6 +31,7 @@ import { MappingSerializer } from '../utils/mapping-serializer';
 import { DataMapperUtil } from '../common/data-mapper-util';
 import { PaddingField } from '../models/document-definition.model';
 import { LookupTableUtil } from '../utils/lookup-table-util';
+import { MappingUtil } from 'utils';
```

Comment:
Better use relative paths. Unfortunately TS paths are not respected when building the package as an npm module (I found this the hard way).

Reply:
actrually didn't need MappingUtil - good catch.